src/TiledArray/algebra/diis.h

```diff
@@ -267,6 +321,10 @@ namespace TiledArray {
         }
       }
 
+      EigenVectorX get_coeffs() { return C_; }
```

Comment:
Assert that C_ is nonempty so that calls to get_coeffs() before calling extrapolate fail. Also, add comments and return by const ref.